packages/plugin-align/src/Align.ts

```diff
@@ -86,9 +86,9 @@ export class Align<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T
 
             // Compute current alignment.
             let currentAlignment: string;
-            if (typeof alignedAncestor?.modifiers.find(Attributes)?.get('style') === 'string') {
+            if (alignedAncestor?.modifiers.get(Attributes).get('style')) {
                 const styles = alignedAncestor.modifiers
-                    .find(Attributes)
+                    .get(Attributes)
```

Comment:
Maybe you should do:
`const attributes = alignedAncestor.modifiers.find(Attributes)` before the `if` then do `if(attributes` to avoid calling `find/get` twice like you're doing now.